hack/teardown.sh

```diff
@@ -5,9 +5,53 @@ source "$(dirname "${BASH_SOURCE[0]}")/lib/__sources__.bash"
 
 set -Eeuo pipefail
 
+declare -a TEST_KN_NAMESPACE
+TEST_KN_NAMESPACE=("serverless-tests" "serverless-tests1" "serverless-tests2" "serverless-tests3" "serverless-tests-mesh")
+#export TEST_KN_NAMESPACE
+
+function teardown_test_data {
```

Comment:
we removed this as of now.. but again the same reason
note: we need this because we are not deleting cluster every time before tests like x86 does. we ran cluster once and then one the same cluster we try multiple times tests